tests/Unit/Domain/Creators/Test_CylindricalBinaryCompactObject.cpp

```diff
@@ -346,6 +346,9 @@ void test_construction(const CylBCO& creator,
     CHECK(excision_sphere_a.center().get(i) == center_objectA.at(i));
     CHECK(excision_sphere_b.center().get(i) == center_objectB.at(i));
   }
+  const auto& block = domain.blocks()[0];
+  CHECK(block.is_time_dependent() == excision_sphere_a.is_time_dependent());
+  CHECK(block.is_time_dependent() == excision_sphere_b.is_time_dependent());
```

Comment:
So I can't do the second thing because the Sphere domain creator has an excision sphere, but it can have a Shape map in the inner shell of blocks. However, we still don't have the proper transition functions to use for inside an excision. So I couldn't add an actual map to the excision sphere. I either had to 1) add an identity map to the excision sphere or 2) just not add a map, so I opted for 2. However, this means that the check that an excision sphere is time dependent if all blocks abutting it are time dependent isn't valid anymore so I got rid of it. We can add it once we figure out the sphere transition stuff.